torchvision/prototype/transforms/_geometry.py

```diff
@@ -389,7 +389,7 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
                 shear_y = float(torch.empty(1).uniform_(self.shear[2], self.shear[3]).item())
 
         shear = (shear_x, shear_y)
-        return dict(angle=angle, translations=translations, scale=scale, shear=shear)
+        return dict(angle=angle, translate=translate, scale=scale, shear=shear)
```

Comment:
Nice catch! Seems like I just copied the name from stable RandomAffine.get_params but unfortunately previous tests didn't revealed that.